lxd/images.go

```diff
@@ -106,7 +106,7 @@ func unpack(file string, path string) error {
 	if err != nil {
 		shared.Debugf("Unpacking failed")
 		shared.Debugf(string(output))
-		return err
+		return fmt.Errorf("Unpack failed, %s.", err)
```

Comment:
Yeah, `err` isn't super helpful, but unfortunately `output` is often massive, like hundreds of lines.  I'll see what type of messages show up if I grab the first ~80 chars of the output.